docs/src/api/linear_solver_routines.md

```diff
@@ -40,8 +40,36 @@ LinSysVecColProjPO
 LinSysVecColProjFO
 ```
 
-## Subsolver Function
+## Block Row Solver
 
 ```@docs
-RLinearAlgebra.rsubsolve!(::Nothing,::AbstractVector,::Tuple,::Int64)
+LinSysBlkRowLQ
 ```
+
+## Block Column Solver
+
+```@docs
+LinSysBlkColGent
+```
+## Solving Routine
+```@docs
+RLinearAlgebra.rsubsolve!
+```
+
+## Gentleman's Solver for Least Squares
```

Comment:
Should we split this into multiple pages?

Reply:
I split this over multiple page by adding a new section to linear solving routines called linear solver routine helpers, this may not be the best spot to place this though.